op-service/txmgr/send_state.go

```diff
@@ -76,7 +72,7 @@ func (s *SendState) ProcessSendError(err error) {
 		s.successFullPublishCount++
 	case errStringMatch(err, core.ErrNonceTooLow):
 		s.nonceTooLowCount++
-	case errStringMatch(err, ErrAlreadyReserved):
+	case errStringMatch(err, txpool.ErrAlreadyReserved):
 		s.alreadyReserved = true
```

Comment:
Other cases here seem to count up the errors by type, while this just counts true/false. Any reason for that?

Reply:
It's not an error we expect to recur beyond the initial (successful) cancellation, so this seemed more natural.  though you're right looking at successFullPublishCount it's a bit inconsistent. Personally I think that one should also be changed to a bool :).     nonceTooLow must be a counter since we actually rely on the value being more than just 0/non-zero.